hasek/logging.py

```diff
@@ -0,0 +1,166 @@
+import logging
```

Comment:
This module feels kinda heavy for what should be a relatively small tool.  Probably leftovers from how Giraffez did logging, maybe could be simplified a bunch.